workspace_analyzer.py

```diff
@@ -248,6 +248,24 @@ def isodd(x):
     inside = points[np.where(isodd(num_intersections))]
     return inside
 
+def alpha_intersection(shape_1, shape_2):
+    """
+    Return a cut down version of shape_1, including only what is in shape_2
+    Args:
+        shape_1: Alpha shape to be trimmed
+        shape_2: Alpha shape to specify trim
+    Returns:
+        new_shape: Shape created via the existing boundaries
+    """
+    inside_shape_1 = inside_alpha_shape(shape_1, shape_2.triangles)
+    inside_shape_2 = inside_alpha_shape(shape_2, shape_1.triangles)
+    inside_shape_1_sz = len(inside_shape_1)
+    inside_shape_2_sz = len(inside_shape_2)
+    new_points = np.zeros((inside_shape_1_sz + inside_shape_2_sz, 3))
+    new_points[0:inside_shape_1_sz] = inside_shape_1
+    new_points[inside_shape_1_sz:] = inside_shape_2
+    new_shape = AlphaShape(new_points, 1.2)
+    return new_shape
```

Comment:
```suggestion
    return AlphaShape(new_points, 1.2)
```
Also, why 1.2 here? Consider constant with comment explaining.

Reply:
The 1.2 (1.98, .5, etc.) is the "Alpha Parameter" which determines which points are included or excluded to create an alpha shape. This is technically an optional parameter, however when the alpha shape optimizes this itself, for large clouds of points it can take many hours. 1.2 is the constant I chose which seems to work for a variety of tested robot sizes, which accepts the densely packed points resulting from workspace analysis. I've added a constant and a comment explaining.